src/ontology/mondo-ingest.Makefile

```diff
@@ -355,23 +358,43 @@ deploy-mondo-ingest:
 
 USE_MONDO_RELEASE=false
 
+# Builds tmp/mondo/ and rebuilds mondo.owl and mondo.sssom.tsv, and stores hash of latest commit of mondo repo main branch in tmp/mondo_repo_built
 tmp/mondo_repo_built:
 	if [ $(USE_MONDO_RELEASE) = true ]; then wget http://purl.obolibrary.org/obo/mondo.owl -O $@; else cd $(TMPDIR) &&\
 		rm -rf ./mondo/ &&\
 		git clone --depth 1 https://github.com/monarch-initiative/mondo &&\
 		cd mondo/src/ontology &&\
 		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
-		cd ../../../../ &&\
-		touch $@; fi
-
-$(TMPDIR)/mondo.sssom.tsv: $(TMPDIR)/mondo_repo_built
-	cp $(TMPDIR)/mondo/src/ontology/mappings/mondo.sssom.tsv $@
-
-$(TMPDIR)/mondo.owl: $(TMPDIR)/mondo_repo_built
-	cp $(TMPDIR)/mondo/src/ontology/mondo.owl $@
-
-reports/mirror_signature-mondo.tsv: tmp/mondo.owl
-	$(ROBOT) query -i $< --query ../sparql/classes.sparql $@
+		latest_hash=$$(git rev-parse origin/master) &&\
+		echo "$$latest_hash" > $@ &&\
+		cp $@ mappings/mondo.sssom.tsv mondo.owl ../../../; fi
+
+# Triggers a refresh of tmp/mondo/ and a rebuild of mondo.owl and mondo.sssom.tsv, only if mondo repo main branch has new commits
+.PHONY: refresh-mondo-clone
+refresh-mondo-clone:
+	@if [ ! -d tmp/mondo ]; then \
+		$(MAKE) tmp/mondo_repo_built -B; \
+	else \
+		current_hash=$$(cat tmp/mondo_repo_built); \
+		cd tmp/mondo; \
+		git fetch origin; \
+		latest_hash=$$(git rev-parse origin/master); \
+		if [ ! "$$current_hash" = "$$latest_hash" ]; then \
+			cd .. && mv mondo mondo-bak && mv mondo_repo_built mondo_repo_built-bak; \
+			cd .. && $(MAKE) tmp/mondo_repo_built -B; \
+			rm -rf tmp/mondo-bak tmp/mondo_repo_built-bak; \
+		fi; \
+	fi
+
+.PHONY: up-to-date-mondo.sssom.tsv
+up-to-date-mondo.sssom.tsv: refresh-mondo-clone
+
+.PHONY: up-to-date-mondo.owl
+up-to-date-mondo.owl: refresh-mondo-clone
```

Comment:
## Add goal `up-to-date-mondo-edit.owl`?
This is a new file that depends on `tmp/mondo/`. See:
- https://github.com/monarch-initiative/mondo-ingest/pull/568#pullrequestreview-2119425130